CHANGES.md

```diff
@@ -52,6 +52,7 @@ To be released.
  -  `Swarm<T>.PreloadAsync()` became to ignore peers with lower tip.  [[#592]]
  -  `KademliaProtocol<T>.RefreshTableAsync()` became to validate only stale
     peers.  [[#568], [#593]]
+ -  Marked `Address` and `HashDigest` as readonly and made some initialization changes.  [[#610]]
```

Comment:
I guess it's okay that we don't expose changes about initialization. 😄

Reply:
Do you mean to remove the line "and made some initialization changes"?

Reply:
Yes.